spring-integration-ip/src/test/java/org/springframework/integration/ip/tcp/connection/TcpNioConnectionTests.java

```diff
@@ -254,6 +254,7 @@ public void testMemoryLeak() throws Exception {
 	}
 
 	@Test
+	@EnabledOnJre(JRE.JAVA_8)
```

Comment:
Would you mind elaborate more for this use-case?

Is something wrong with reflection?

Reply:
Invalid reflection at line 276 - no such field in JDK10.